bin/crystal

```diff
@@ -142,7 +142,7 @@ export CRYSTAL_PATH=$CRYSTAL_ROOT/src:lib
 export CRYSTAL_HAS_WRAPPER=true
 
 if [ -x "$CRYSTAL_DIR/crystal" ]; then
-  __warning_msg "Using compiled compiler at \`.build/crystal'"
+  __warning_msg "Using compiled compiler at $CRYSTAL_DIR/crystal"
```

Comment:
Could we print the full path only when `pwd` doesn't start with `$CRYSTAL_ROOT`?

Reply:
That did work, thank you!  Now it says:

```
$ ./bin/crystal 
Using compiled compiler at .build/crystal

$ ./dev/crystal/bin/crystal 
Using compiled compiler at dev/crystal/.build/crystal
```
and the full path if you're not in a relative directory.

Reply:
@Sija why the confused react?

@rdp thoughts on adding `./`before the path?

Reply:
I was confused as to why not leave the full path regardless?